frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -413,6 +402,19 @@ export const PersistentVolumeClaimsPage = (props) => {
           },
         };
 
+  const { t } = useTranslation();
```

Comment:
Move this to the top of the function. All hooks need to be called in the same order each time, so it's better to keep them at the top. Otherwise an early return would prevent the hook from being called.

Reply:
> Move this to the top of the function. All hooks need to be called in the same order each time, so it's better to keep them at the top. Otherwise an early return would prevent the hook from being called.

This comment is still pending.

Reply:
Updated